### PR TITLE
feat: Add Uno Platform Sandbox and Quickstart links

### PR DESCRIPTION
**Pull request by agneszitte**
Add Uno Platform Sandbox and Quickstart links
Related PR: [feat: Add ScottPlot Uno Platform QuickStart](https://github.com/ScottPlot/ScottPlot.NET/pull/40#top)

### CODE REVIEW

File: src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    "Sandbox.Unplatform (WinAppSDK Packaged)": {
+      "commandName": "MsixPackage",
+      "compatibleTargetFramework": "windows"
+    },
+    "Sandbox.Unplatform (Desktop)": {
+      "commandName": "Project",
+      "compatibleTargetFramework": "desktop"
+    },
+    "Sandbox.Unplatform (Desktop WSL2)": {
```
Comment (agneszitte) [inline src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json:43]: I have tested all platforms. However, I am currently unable to launch **Desktop WSL2 with the Sandbox** due to a space in the path. 

For related details on the Uno Platform side, please refer to:
- This comment: [unoplatform/private#479 (comment).](https://github.com/unoplatform/private/issues/479#issuecomment-2221864398)).
- The current documentation: https://platform.uno/docs/articles/common-issues-vs2022.html#my-application-does-not-start-under-wsl

I am uncertain about the best approach to address this issue. Here are a few options:

1. Change the folder name to remove the space and fix the issue.
2. Leave a comment in MainPage.xaml or another relevant file explaining this issue.
3. Do you have any other suggestions please?

image

Thanks in advance for the feedback

**Comment by swharden** [comment]
Hi @agneszitte, thanks so much for this PR! I look forward to merging it soon, but I'm travelling for the next several days so it may take me until next week before I can get started.

One thing I notice is that there are a few "dot files" in this commit. Specifically `.vscode` and `.run` folders and `.vsconfig` and `.editorconfig` files. I wanted to double-check with you whether this was intentional before this gets merged into the main branch. Thanks for your input!

**Comment by swharden** [comment]
Regarding the common weakness identified by CodeQL, more information is on https://codeql.github.com/codeql-query-help/csharp/cs-web-missing-global-error-handler/ which provides recommendations for how to edit `web.config` to be a little more secure.

I recognize that having this code in the sandbox app is low risk in itself, but it would be great if we could implement a fix to make this a little more secure by default in case it gets copy/pasted into a production environment.

**Comment by agneszitte** [comment]
> Regarding the common weakness identified by CodeQL, more information is on [codeql.github.com/codeql-query-help/csharp/cs-web-missing-global-error-handler](https://codeql.github.com/codeql-query-help/csharp/cs-web-missing-global-error-handler/) which provides recommendations for how to edit `web.config` to be a little more secure.
> 
> I recognize that having this code in the sandbox app is low risk in itself, but it would be great if we could implement a fix to make this a little more secure by default in case it gets copy/pasted into a production environment.

@swharden  I have removed the `web.config` default file in the sandbox app. It’s unnecessary for local tests for WebAssembly and primarily used for deployments on IIS. When creating a new Uno Platform project, this file is added by default but can be adjusted as needed to enhance security.

**Comment by agneszitte** [comment]
> Hi @agneszitte, thanks so much for this PR! I look forward to merging it soon, but I'm travelling for the next several days so it may take me until next week before I can get started.
> 
> One thing I notice is that there are a few "dot files" in this commit. Specifically `.vscode` and `.run` folders and `.vsconfig` and `.editorconfig` files. I wanted to double-check with you whether this was intentional before this gets merged into the main branch. Thanks for your input!

@swharden I removed them as well. They are added by default with a new Uno Platform project but not necessary in this case.

**Comment by agneszitte** [comment]
Hi @swharden, let me know if you need anything else for this PR please

**Comment by swharden** [comment]
Hi @agneszitte, thanks again for this PR! It took me longer than expected to get to it, but I look forward to merging this today, publishing a NuGet package, and updated the docs on the website.

> I am currently unable to launch Desktop WSL2 with the Sandbox with Visual Studio for this solution due to a space in the path.

> https://platform.uno/docs/articles/common-issues-vs2022.html#my-application-does-not-start-under-wsl

Because a proper fix would involve renaming a lot of important folders, I'm satisfied leaving it as it in the sandbox app. The important thing is that users can create new Uno Platform projects that use ScottPlot by following the QuickStart guide on the website. As long as the docs are in a good place, I think most people will not encounter this issue.

File: src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json (hunk 1 above)
Comment (swharden) [reply src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json:43]: https://github.com/ScottPlot/ScottPlot/pull/4123#issuecomment-2311250146

**Comment by agneszitte** [comment]
Thanks a lot, @swharden for merging the changes here <3

**Pull request merged**